src/main/java/com/mongodb/jdbc/MongoFunctions.java

```diff
@@ -79,17 +93,85 @@ public static MongoFunctions getInstance() {
             instance =
                     new MongoFunctions(
                             new MongoFunction[] {
+                                new MongoFunction(
+                                        ABS,
+                                        BsonTypeInfo.BSON_INT.getBsonName(),
+                                        "returns the absolute value of the given number.",
+                                        new String[] {BsonTypeInfo.BSON_INT.getBsonName()},
+                                        FunctionCategory.NUM_FUNC),
+                                new MongoFunction(
+                                        ABS,
+                                        BsonTypeInfo.BSON_LONG.getBsonName(),
+                                        "returns the absolute value of the given number.",
+                                        new String[] {BsonTypeInfo.BSON_LONG.getBsonName()},
+                                        FunctionCategory.NUM_FUNC),
+                                new MongoFunction(
+                                        ABS,
+                                        BsonTypeInfo.BSON_DOUBLE.getBsonName(),
+                                        "returns the absolute value of the given number.",
+                                        new String[] {BsonTypeInfo.BSON_DOUBLE.getBsonName()},
+                                        FunctionCategory.NUM_FUNC),
+                                new MongoFunction(
+                                        ABS,
+                                        BsonTypeInfo.BSON_DECIMAL.getBsonName(),
+                                        "returns the absolute value of the given number.",
+                                        new String[] {BsonTypeInfo.BSON_DECIMAL.getBsonName()},
+                                        FunctionCategory.NUM_FUNC),
                                 new MongoFunction(
                                         "BIT_LENGTH",
                                         BsonTypeInfo.BSON_LONG.getBsonName(),
                                         "returns length of string in bits",
                                         new String[] {BsonTypeInfo.BSON_STRING.getBsonName()}),
+                                new MongoFunction(
```

Comment:
This looks to be missing a `FunctionCategory.NUM_FUNC` as a last parameter.  Same with the `LOG` and `POW` categories.

Reply:
The JDBC spec has a "CEILING" function but not a "CEIL" function. From previous PRs and conversations, I remember us deciding not to mark anything with JDBC labels (such as "numeric", "system", etc) unless they were exact matches with the spec. Even though our "CEIL" behaves the same as their "CEILING", the name is different so I omitted the label intentionally.

Reply:
Ah that makes sense!  Thanks for clarifying.